apps/rule-manager/app/service/RuleTesting.scala

```diff
@@ -0,0 +1,59 @@
+package service
+
+import play.api.libs.ws.WSClient
+import com.gu.typerighter.lib.{HMACClient, JsonHelpers, Loggable}
+import com.gu.typerighter.model.{CheckSingleRule, CheckSingleRuleResult, Document}
+import db.DbRuleDraft
+import play.api.libs.json.{JsError, JsSuccess, Json}
+
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
+
+class RuleTesting(
+    ws: WSClient,
+    hmacClient: HMACClient,
+    checkerUrl: String
+) extends Loggable {
+
+  /** Test a rule against the given list of documents. Return a list of matches from the checker
+    * service endpoint as a stream of json-seq records.
+    */
+  def testRule(rule: DbRuleDraft, documents: List[Document])(implicit ec: ExecutionContext) = {
+    val liveRule = rule.toLive("placeholder")
```

Comment:
Heh, a little strange but I understand why!

Reply:
I guess we could add a method `draftDbRuleToCheckerRule`, but it's not a blocker (a new method might not be as DRY).

Reply:
Yeah, it's a bit of a pain, this. Be a nice refactor down the line.